android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -801,10 +801,23 @@ public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest request
 
     @Override
     public void onReceivedSslError(final WebView webView, final SslErrorHandler handler, final SslError error) {
+        // WebView.getUrl() will return the top-level window URL.
+        // If a top-level navigation triggers this error handler, the top-level URL will be the failing URL (not the current URL).
+        // This is desired behavior. We later use these values to determine whether the request is a top-level navigation or a subresource request.
+        String topWindowUrl = webView.getUrl();
+        String failingUrl = error.getUrl();
+        
+        // Cancel request after obtaining top-level URL.
+        // If request is cancelled before obtaining top-level URL, undesired behavior may occur.
+        // Undesired behavior: Return value of WebView.getUrl() may be the current URL instead of the failing URL.
         handler.cancel();
 
+        if (!topWindowUrl.equalsIgnoreCase(failingUrl)) {
+          // If error is not due to top-level navigation, then do not call onReceivedError()
+          return;
```

Comment:
I assume the resulting behavior is just a blank space where the nested content would have been?

It would be nice to have _some_ indication of what happened. If not via new event, perhaps at least just log a message here so it can be picked up by logcat?

Reply:
The behavior is controlled by the Android WebView. For visible resources (e.g. iframes), I believe it would be either blank or show an embedded SSL error page, but this may vary based on Android WebView version. For non-visible resources (e.g. scripts), nothing is visible.

I'll add quick log message.